app/models/imported_clients_csv.rb

```diff
@@ -16,6 +21,12 @@ def initialize(attributes = {})
     @clients = []
   end
 
+  # Validate file content before creating record
+  def self.validate_file_content(file_content, claimed_content_type = nil)
+    allowed_types = ['text/plain', 'text/csv', 'application/csv']
```

Comment:
I guess windows uses 'application/vnd.ms-excel' for csv under certain circumstances 
```suggestion
    allowed_types = ['text/plain', 'text/csv', 'application/csv', 'application/vnd.ms-excel']
```